docs/.vitepress/sidebars/base.ts

```diff
@@ -0,0 +1,242 @@
+            items: getMarkdownFiles('/products/smart-contract-toolkit'),
+            collapsed: true,
+        },
+        {
+            text: 'Ultratest',
+            items: getMarkdownFiles('/products/ultratest'),
+            collapsed: true,
+        },
+        {
+            text: 'Uniq Discord Bot',
+            items: getMarkdownFiles('/products/uniq-discord-bot'),
+            collapsed: true,
+        },
+    ],
+    // ################
+    // blockchain/general
+    // ################
+    '/blockchain/general': [
+        {
+            text: 'Blockchain General',
+            items: [
+                {
+                    text: 'What is Ultra?',
+                    link: '/blockchain/general/index',
+                },
+                {
+                    text: 'Useful Links',
+                    link: '/blockchain/general/links',
+                },
+            ],
+        },
+        {
+            text: 'Ultra Blockchain',
+            items: getMarkdownFiles('/blockchain/general/antelope-ultra'),
+            collapsed: true,
+        },
+        {
+            text: 'Tools',
+            items: getMarkdownFiles('/blockchain/general/tools'),
+            collapsed: true,
+        },
+    ],
+    // ################
+    // blockchain/block-producers
+    // ################
+    '/blockchain/block-producers': [
+        {
+            text: 'Block Producers',
+            items: [
+                {
+                    text: 'What is a Block Producer?',
+                    link: '/blockchain/block-producers/index',
+                },
+            ],
+        },
+        {
+            text: 'Infrastructure',
+            items: getMarkdownFiles('/blockchain/block-producers/infrastructure'),
+            collapsed: true,
+        },
+        {
+            text: 'Launch Procedures',
+            items: getMarkdownFiles('/blockchain/block-producers/launch-procedures'),
+            collapsed: true,
+        },
+        {
+            text: 'Maintenance',
+            items: getMarkdownFiles('/blockchain/block-producers/maintenance'),
+            collapsed: true,
+        },
+    ],
+    // ################
+    // blockchain/contracts
+    // ################
+    '/blockchain/contracts': [
+        {
+            text: 'Ultra Contracts',
+            items: [
+                {
+                    text: 'What is Ultra?',
+                    link: '/blockchain/contracts/index',
+                },
+            ],
+        },
+        {
+            text: 'System Contract',
+            items: [
+                ...getMarkdownFiles('/blockchain/contracts/system-contract'),
+                {
+                    text: 'Actions',
+                    items: getMarkdownFiles('/blockchain/contracts/system-contract/system-actions'),
+                    collapsed: true,
+                },
+            ],
+            collapsed: true,
+        },
+        {
+            text: 'NFT Contract',
```

Comment:
We said that in ultra we call all NFTs -> 'Uniqs'.  should this one also change? like **Uniq Contract**?

Reply:
We call NFTs as Uniqs, yes, but the contract itself is `eosio.nft.ft` - NFT contract